### PR TITLE
[MB-14343] Playwright tests for SC Pro-gear and Expenses Review

### PR DESCRIPTION
**Pull request by bellex3**
## Summary
Added E2E Playwright tests for Service Counselor review of Pro-gear tickets and Expenses. Tests that SC can: 
**Pro-gear**
review, approve/reject, and edit pro-gear tickets 

**Expenses**
review, approve/reject/exclude, and edit expenses

## Setup to Run Your Code

**To run Playwright tests locally:** 

1. Need to run both server and client. `make server_run` and `make office_client_run`.
2. Run the following commands: 

**For pro-gear tests**

```
npx playwright test playwright/tests/office/servicescounseling/servicesCounselingProGearTickets.spec.js --trace="on"
```

**For expenses tests**
```
npx playwright test playwright/tests/office/servicescounseling/servicesCounselingMovingExpenses.spec.js --trace="on"
```

You can also run: 
```
./scripts/run-e2e-test --trace=on playwright/tests/office/servicescounseling/servicesCounselingProGearTickets.spec.js
```
```
./scripts/run-e2e-test --trace=on playwright/tests/office/servicescounseling/servicesCounselingMovingExpenses.spec.js
```

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Screenshots
Screenshot 2023-03-16 at 1 04 00 PM

[MB-14343]: https://dp3.atlassian.net/browse/MB-14343?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: playwright/tests/office/servicescounseling/servicesCounselingMovingExpenses.spec.js
Comment (pearl-truss) [inline playwright/tests/office/servicescounseling/servicesCounselingMovingExpenses.spec.js:1]: **Praise:** I appreciate the comments, makes it easy to know where in the flow the test is!

File: pkg/testdatagen/testharness/make_move.go [hunk 1]
```diff
@@ -2441,7 +2441,7 @@ func MakeApprovedMoveWithPPMProgearWeightTicket(appCtx appcontext.AppContext) mo
 		ServiceMember: move.Orders.ServiceMember,
 	}
 	testdatagen.MakeWeightTicket(appCtx.DB(), ppmCloseoutAssertions)
-	testdatagen.MakeMovingExpense(appCtx.DB(), ppmCloseoutAssertions)
+	// testdatagen.MakeMovingExpense(appCtx.DB(), ppmCloseoutAssertions)
```
Comment (pearl-truss) [inline pkg/testdatagen/testharness/make_move.go:2444]: **Nit:** I think this could be removed

**Review by pearl-truss — COMMENTED** [review]
Good job and it's interesting seeing a new playwright test! 

I think the happy path test are definitely covered here but I think I noticed the edit test are missing. There's some outlined in the AC for [this ticket](https://dp3.atlassian.net/browse/MB-14343)

**Comment by bellex3** [comment]
> Good job and it's interesting seeing a new playwright test!
> 
> I think the happy path test are definitely covered here but I think I noticed the edit test are missing. There's some outlined in the AC for [this ticket](https://dp3.atlassian.net/browse/MB-14343)

Inquired about this offline, and confirmed to match the pattern that is already implemented in the weight tickets playwright test. [Slack convo](https://ustcdp3.slack.com/archives/CP6F568DC/p1679077335564889) for reference.

File: pkg/testdatagen/testharness/make_move.go (hunk 1 above)
Comment (bellex3) [reply pkg/testdatagen/testharness/make_move.go]: done

File: playwright/tests/office/servicescounseling/servicesCounselingTestFixture.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ export class ServiceCounselorPage extends OfficePage {
       await expect(this.page.getByRole('heading', { name: 'Review trip 1', level: 3 })).toBeVisible();
     },
     reviewDocumentsConfirmation: async () => {
-      await expect(this.page.getByRole('heading', { name: 'Send to customer?', level: 3 })).toBeVisible();
+      await expect(this.page.getByRole('heading', { name: 'Send to customer?' })).toBeVisible();
```
Comment (scottmries) [inline playwright/tests/office/servicescounseling/servicesCounselingTestFixture.js:58]:
```suggestion
      await expect(this.page.getByRole('heading', { name: 'Send to customer?', level: 3 })).toBeVisible();
```

Comment (scottmries) [reply playwright/tests/office/servicescounseling/servicesCounselingTestFixture.js:58]: Issue: I think we want to enforce this heading level.

**Review by scottmries — COMMENTED** [review]
Great work, @bellex3!

praise: I second Pearl's praise, those comments are very helpful in the tests.

I just left one suggestion. Once that's resolved, I think we'll be good to approve.

File: playwright/tests/office/servicescounseling/servicesCounselingTestFixture.js (hunk 2 above)
Comment (bellex3) [reply playwright/tests/office/servicescounseling/servicesCounselingTestFixture.js]: done

**Review by scottmries — APPROVED** [review]

**Pull request merged**